python/hail/expr/ast.py

```diff
@@ -199,6 +199,21 @@ def to_hql(self):
         return "{start}:{end}".format(start=self.start.to_hql() if self.start else '',
                                       end=self.stop.to_hql() if self.stop else '')
 
+class Let(AST):
```

Comment:
Seems like it might be confusing calling it `Let` here, and `bind` everywhere else. Did you have a reason for the mismatch?